ts/core/MmlTree/SerializedMmlVisitor.ts

```diff
@@ -203,6 +203,7 @@ export class SerializedMmlVisitor extends MmlVisitor {
     node.getProperty('variantForm') && this.setDataAttribute(data, 'alternate', '1');
     node.getProperty('pseudoscript') && this.setDataAttribute(data, 'pseudoscript', 'true');
     node.getProperty('mathaccent') && this.setDataAttribute(data, 'accent', 'true');
+    node.getProperty('autoOP') === false && this.setDataAttribute(data, 'auto-op', 'false');
```

Comment:
This branch now has a merge conflict due to this line (since https://github.com/mathjax/MathJax-src/pull/654 was merged and removed the preceding line). 

Could you let me know whether this line should stay?

Reply:
Yes, this line should stay.

Reply:
But the preceding line should go.  Sorry if I misunderstood the question.

Reply:
Thanks, Davide. All clear.